lib/node_modules/@stdlib/stats/base/dnanmeanpw/README.md

```diff
@@ -59,28 +59,25 @@ Computes the [arithmetic mean][arithmetic-mean] of a double-precision floating-p
 var Float64Array = require( '@stdlib/array/float64' );
 
 var x = new Float64Array( [ 1.0, -2.0, NaN, 2.0 ] );
-var N = x.length;
 
-var v = dnanmeanpw( N, x, 1 );
+var v = dnanmeanpw( x.length, x, 1 );
 // returns ~0.3333
 ```
 
 The function has the following parameters:
 
 -   **N**: number of indexed elements.
 -   **x**: input [`Float64Array`][@stdlib/array/float64].
--   **stride**: index increment for `x`.
+-   **strideX**: stride length.
 
-The `N` and `stride` parameters determine which elements in `x` are accessed at runtime. For example, to compute the [arithmetic mean][arithmetic-mean] of every other element in `x`,
+The `N` and stride parameters determine which elements in the strided array are accessed at runtime. For example, to compute the [arithmetic mean][arithmetic-mean] of every other element:
 
 ```javascript
 var Float64Array = require( '@stdlib/array/float64' );
-var floor = require( '@stdlib/math/base/special/floor' );
 
 var x = new Float64Array( [ 1.0, 2.0, 2.0, -7.0, -2.0, 3.0, 4.0, 2.0, NaN ] );
-var N = floor( x.length / 2 );
 
-var v = dnanmeanpw( N, x, 2 );
+var v = dnanmeanpw( 4, x, 2 );
```

Comment:
```suggestion
var v = dnanmeanpw( 5, x, 2 );
```

While `4` is correct, it is not the intent. The point is to have a `NaN` value in the calculation. If `N = 4`, then computation stops at `4.0`. Probably worthwhile looking over the examples once more to sanity check.

Reply:
It is this sort of thing which motivates why we've been removing the use of `floor`, as it can mask documentation and test bugs. Better to be explicit about the number of elements over which to perform computation.